azure-pipelines.yaml

```diff
@@ -191,6 +190,11 @@ variables:
   - template: vars/input-variables.yaml@cnp-azuredevops-libraries
   - name: timeoutInMinutes
     value: 60
+  - name: finalAction
+    ${{ if and(eq(variables['isMain'], true), eq(variables['isAutoTriggered'], true)) }}:
+      value: 'apply'
```

Comment:
Is that correct? I could run a plan of main branch

Reply:
do we allow runing a plan of main branch that is not auto triggered?